iscript/src/iscript/data/i_task_schema.json

```diff
@@ -99,6 +101,35 @@
             "entitlements-url": {
               "type": "string"
             },
+            "signing-config": {
```

Comment:
Can we find another name for this, please? The entire schema is already arguably a "signing config" of sorts. If this is specifically only for hardened signing, perhaps something like `hardening-config`? (I'm open to other ideas - just wanted to throw something out there.)